### PR TITLE
Remote mute/unmute

### PR DESCRIPTION
**Pull request by hiroshihorie**

### CODE REVIEW

File: Sources/LiveKit/classes/Room.swift [hunk 1]
```diff
@@ -276,6 +276,12 @@ extension Room: RTCEngineDelegate {
         participant.delegate?.didReceive(data: packet.payload, participant: participant)
     }
 
+    func didRemoteMuteChange(trackSid: String, muted: Bool) {
```
Comment (davidzhao) [inline Sources/LiveKit/classes/Room.swift:279]: nit

```suggestion
    func remoteMuteDidChange(trackSid: String, muted: Bool) {
```

File: Sources/LiveKit/protocols/RTCEngineDelegate.swift [hunk 2]
```diff
@@ -18,4 +18,5 @@ protocol RTCEngineDelegate: AnyObject {
     func didDisconnect()
     func didFailToConnect(error: Error)
     func didReceive(packet: Livekit_UserPacket, kind: Livekit_DataPacket.Kind)
+    func didRemoteMuteChange(trackSid: String, muted: Bool)
```
Comment (davidzhao) [inline Sources/LiveKit/protocols/RTCEngineDelegate.swift:21]:
```suggestion
    func remoteMuteDidChange(trackSid: String, muted: Bool)
```

**Review by davidzhao — APPROVED** [review]
looks good! just a nit

File: Sources/LiveKit/protocols/RTCEngineDelegate.swift (hunk 2 above)
Comment (hiroshihorie) [reply Sources/LiveKit/protocols/RTCEngineDelegate.swift:21]: Yea I aligned it with the other 'did's above haha but I'll change 😁

**Pull request merged**